LSL/liblsl/src/sample.h

```diff
@@ -42,6 +42,7 @@ namespace lsl {
 		class factory;					// samples can only be created by the factory
 		double timestamp;				// time-stamp of the sample
 		bool pushthrough;				// whether the sample shall be buffered or pushed through
+		bool is_managed;
```

Comment:
Does this work with the current protocol version? The client getting the sample doesn't care about the memory management on the sender side so we don't need to send it, but we need to keep track of it somewhere.